portfolio-project-matching-app/tailwind.config.js

```diff
@@ -1,8 +1,27 @@
+const defaultTheme = require('tailwindcss/defaultTheme')
+
 module.exports = {
   purge: ['./pages/**/*.{js,ts,jsx,tsx}', './components/**/*.{js,ts,jsx,tsx}'],
   darkMode: false, // or 'media' or 'class'
   theme: {
-    extend: {},
+    extend: {
+      screens: {
+        'xs': '420px',
+        ...defaultTheme.screens,
```

Comment:
I was having fun adding an extra break point for super small screens, might come in handy when reworking things in favor of responsiveness. I don't think I ended up using this in the end.